src/common/personalizedDigest.ts

```diff
@@ -216,6 +217,9 @@ export const getPersonalizedDigestEmailPayload = async ({
     blocked_tags: feedConfig.blockedTags,
     blocked_sources: feedConfig.excludeSources,
     feed_config_name: feature.feedConfig,
+    source_types: baseFeedConfig.source_types.filter(
+      (el) => !feedConfig.excludeSourceTypes.includes(el),
+    ),
```

Comment:
Digest config supports it, I know in the past we had to add/enabled specifically, maybe check with Vas?

Reply:
Yeah checked, he's adding it to all:
https://dailydotdev.slack.com/archives/C04UN615UAF/p1727855255223979